pkg/system/system.go

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package system
+
+import (
+	"encoding/json"
+	"net"
+	"net/http"
+	"os"
+
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/nydus-snapshotter/pkg/daemon"
+	"github.com/containerd/nydus-snapshotter/pkg/manager"
+	"github.com/containerd/nydus-snapshotter/pkg/metrics/exporter"
+	"github.com/gorilla/mux"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+const (
```

Comment:
We'd better mark all these endpoints as `expirimental`, so that we have a chance to adjust the response JSON format.

Reply:
marked